.changeset/tasty-goats-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/styled': patch
+---
+
+Fix `props.theme` type in styled component interpolation, this was `Theme | undefined` instead of `Theme`.
```

Comment:
```suggestion
Fix `props.theme` type in styled component interpolation being optional (`Theme | undefined`) for components without `AdditionalProps` being specified.
```

Reply:
I've wanted to commit this suggestion through the GitHub interface - but it seems like I don't have sufficient permissions to push to your fork.

You can check how to allow this for the future PRs here - https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork . At the moment just commit this suggestion with a click on a button.

Reply:
![image](https://user-images.githubusercontent.com/1761469/72069048-9280c180-32e6-11ea-8aa9-10844b56e46f.png)

This was already set, you are maybe not considered as maintainer 🙄

Reply:
That's weird then, I'm an organization member with admin rights here - also I've been successfully committing such suggestions in the past in other PRs. Maybe GitHub has some hiccup 🤷‍♂